docs/SELENIUM-GRID.md

```diff
@@ -0,0 +1,30 @@
+#Running ATH tests against Selenium Grid
+
+This feature enables us to test ATH against Selenium Grid
+
+##Requirements
+* JENKINS_URL must be accessible for the Selenium hub
+* If running against existing Jenkins instance using Jenkins Under Test (JUT) =EXISTING, A running Jenkins instance with pre-installed [Form-Element-Path plugin](https://wiki.jenkins-ci.org/display/JENKINS/Form+Element+Path+Plugin)
```

Comment:
I did not find this limitation in CONTROLLER.md, hence I added it. I guess I'll remove it from here then.